config/database-encryption.php

```diff
@@ -3,7 +3,7 @@
  * src/config/database-encryption.php.
  *
  * @author      Austin Heap <me@austinheap.com>
- * @version     v0.2.1
+ * @version     v0.2.2
```

Comment:
This is *at least* a major unreleased change (v0.3.0), not a minor one.
```suggestion
 * @version     v0.3.0
```